src/enemy_skills.ts

```diff
@@ -1035,7 +1035,7 @@ const skyfall: EnemySkillEffect = {
 
   addMechanic: (mechanic, { skillArgs }) => {
     const colors = idxsFromBits(skillArgs[0]);
-    if (colors.includes(Attribute.POISON) || colors.includes(Attribute.MORTAL_POSION)) {
+    if (colors.includes(Attribute.POISON) || colors.includes(Attribute.MORTAL_POISON)) {

```

Comment:
Question: Why did all of these change?

Reply:
I fixed a typo: it previously was spelled POSION, with the S and I interposed, and I corrected it to POISON.